services/director-v2/src/simcore_service_director_v2/models/domains/comp_tasks.py

```diff
@@ -24,16 +24,31 @@
 class Image(BaseModel):
     name: str = Field(..., regex=KEY_RE)
     tag: str = Field(..., regex=VERSION_RE)
-    requires_gpu: Optional[bool] = Field(None, deprecated=True)
-    requires_mpi: Optional[bool] = Field(None, deprecated=True)
+
+    requires_gpu: Optional[bool] = Field(
+        None, deprecated=True, description="Use instead node_requirements"
+    )
+    requires_mpi: Optional[bool] = Field(
+        None, deprecated=True, description="Use instead node_requirements"
```

Comment:
is there any necessity of keeping both?

Reply:
also what is the meaning of having node_requirements defaults to None? it should at least have CPU right? before since there was no requires_cpu entry it was implicit. now it should be explicit.

Reply:
Those were already there and marked as deprecated. 

The validator below provides an automatic migration and  guarantees backward compatibility. Check the comment in the validator ``def migrate_*``